docs/how-tos/using-events-within-edx-platform.rst

```diff
@@ -0,0 +1,113 @@
+Using Open edX Events within edx-platform
```

Comment:
note to author: consider generalizing and not use edx-platform-specific instructions. Maybe as an example